src/Forcings/forcing.jl

```diff
@@ -160,3 +160,16 @@ function Forcing(func; parameters=nothing, field_dependencies=(), discrete_form=
         return ContinuousForcing(func; parameters=parameters, field_dependencies=field_dependencies)
     end
 end
+
+# Support the case that forcing data is loaded in a 3D array:
+@inline array_forcing_func(i, j, k, grid, clock, fields, a) = @inbounds a[i, j, k]
+
+"""
+    Forcing(array::AbstractArray)
+
+Return a `Forcing` by `array`, which can be added to the tendency of an Oceananigans model field.
+
+Forcing is computed by calling `array[i, j, k]`, so `array` must be 3D with `size(grid)`.
```

Comment:
what if this is forcing for, e.g., $v$ on a Bounded `y` dimension? then does the array need to be of size `(grid.Nx, grid.Ny+1, grid.Nz)`?

Reply:
hmm good question and no, because the tendencies are always computed from 1:grid.Nx, etc, regardless of topology.

The underlying reason is that i=1 and i=grid.Nx+1 are determined by boundary conditions for wall normal velocities (they are not prognostic). For `Bounded`, we do extra computation at `i=1` for bounded velocities (we simply haven't bothered to make that optimization...)

Reply:
okie, great!